dev/com.ibm.ws.microprofile.config.1.4/src/com/ibm/ws/microprofile/config14/impl/Config14Impl.java

```diff
@@ -135,6 +144,18 @@ private SourcedValue getRawValue(String key) {
         return raw;
     }
 
+    @Override
+    public void close() {
```

Comment:
Does the `convertedValueCache` need clearing?

Reply:
No. Potentially that could be an improvement, but this whole class should be GC'd shortly after `close()` is called.

The only reason for closing `TimedCache` is to cancel any pending invalidation task.